app/forms/search_form.rb

```diff
@@ -26,6 +27,10 @@ def caseworkers
     [show_all] + User.order(:last_name, :first_name).map { Option.new(_1.id, _1.display_name) }
   end
 
+  def claim_values
+    [show_all] + %i[high_value].map { Option.new(_1, I18n.t("search.claim_values.#{_1}")) }
```

Comment:
Why is this mapping over an array with 1 value?

Reply:
<img width="570" alt="image" src="https://github.com/user-attachments/assets/ee23d0d1-39cf-47b1-bd48-d679a4398581" />

it's not just mapping high_value, it's mapping high_value + anything in [show_all], the + operation happens before the map

Reply:
Oh high_value is a function, I thought it was a symbol

Reply:
@ivanELEC that doesn't show what you think it shows. Try `[1,2,3] + [4,5].map { _1 * 2 }`. If you were right you'd see `[2,4,6,8,10]`, but you'll actually see `[1,2,3,8,10]`. Ellis is right, the `map` is only being applied to `:high_value`, at which point you might as well just do `[show_all, Option.new(:high_value, I18n.t("search.claim_values.high_value")) }`

Reply:
thanks for keeping me honest there - looks like two wrongs don't make a right after all (even if it looks like they do) - will update to what you suggested as that's simpler to read